test/fixtures/rules/std_rule_cases/L053.yml

```diff
@@ -0,0 +1,67 @@
+rule: L053
+
+test_pass_no_outer_brackets:
+  pass_str: |
+    SELECT foo
+    FROM bar
+
+test_fail_outer_brackets:
+  fail_str: |
+    (SELECT foo
+    FROM bar)
+  fix_str: |
+    SELECT foo
+    FROM bar
+
+test_fail_outer_brackets_inner_subquery:
+  fail_str: |
+    (SELECT foo
+    FROM (select * from bar))
+  fix_str: |
+    SELECT foo
+    FROM (select * from bar)
+
+test_pass_set_statement_brackets:
+  pass_str: |
+    (SELECT 1)
```

Comment:
It's tempting to say the rule should remove these parentheses as well. Thoughts?

Reply:
@barrywhart I think we could go either way on this.

This might be an example of a top level query (it doesn't parse in sqlfluff but is valid in actual sql) that would be covered by this rule.
```SQL
(
(SELECT 1)
UNION
(SELECT 1)
)
```

if we were to cover the lower-level selects as well (which i agree is a nice to have) we would also have to cover things like
```SQL
SELECT *
FROM (
    (SELECT 1)
    UNION
    (SELECT 1)
)
```

I think if we were to do that we would have to start including a bunch of conditions to the rule like 
```
- if we find brackets whose direct parent is a set operator then remove
- if we find brackets whose direct parent is a begin/end statement then remove etc
```
Which we can definitely do, but maybe we consider those as an enhancement at a later date to this rule given the rule currently fulfils its stated aim and it would just be a matter of adding additional conditions?

Reply:
are there other cases that you would want removed?

Reply:
My view is this is out of scope:

```sql
(SELECT 1)
UNION
(SELECT 1)
```

These are not top level brackets. And it's going to get much more complicated if we start trying to include those.

Plus the above example is good practice in a lot of cases to clearly differentiate between the two sub-clauses.

This SQL, however, has no value in the brackets - and that was what we were trying to address (or at least that was my understanding):

```sql
(SELECT 1)
```

Reply:
exactly, think thats the pragmatic aproach 👍

Reply:
Sounds good to me! I hope I'm not being a nuisance -- sometimes I ask questions just to get people's thoughts, whether or not I have a strong opinion myself. As you say, scope creep can be a real problem -- often it's better to get a first version and enhance later as needed. There's plenty of other important work to do on the project, I'm sure.

Thanks for the excellent work -- writing rules is one of the trickiest areas of SQLFluff to learn.

Reply:
@barrywhart It's honestly great, the feedback is super insightful and having the discussions in the PR means we can refer back to this for future developments/issues 😄 
Thanks!